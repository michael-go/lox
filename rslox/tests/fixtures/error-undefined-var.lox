no_such_var = 42;
