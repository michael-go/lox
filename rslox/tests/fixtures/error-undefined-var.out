# exit code: 70
# stdout:

# stderr:
Undefined variable 'no_such_var'.
[line 1] in <script>

